meson.build

```
@@ -1272,22 +1309,13 @@ if host_machine.system() == 'linux'
    # TODO: For the moment we are only producing an English man page.  It would not be huge effort to produce them for
    # non-English locales, so we should do that at some point.
    #
-   pandocResult = run_command(
-      'pandoc',
-      join_paths(meson.current_build_dir(), 'manpage.1.md'),
-      '--verbose',
-      '-t', 'man',
-      '-s',
-      '-o', join_paths(meson.current_build_dir(), projectName + '.1'),
+   generatedManPage = configure_file(
+      input: manPageMarkdown,
+      output: projectName + '.1',
       capture : true,
-      check : true
+      command: ['pandoc', manPageMarkdown, '--verbose', '-t', 'man', '-s'],
```

```suggestion
      command: ['pandoc', '@INPUT@', '--verbose', '-t', 'man', '-s'],
```

Instead of repeating manPageMarkdown, you can have Meson format the input location into the command.